web5-wallet/package.json

```diff
@@ -19,7 +19,7 @@
     "@decentralized-identity/ion-tools": "1.0.3",
     "@js-temporal/polyfill": "0.4.3",
     "@noble/ed25519": "1.6.0",
-    "@tbd54566975/dwn-sdk-js": "0.0.16",
+    "@tbd54566975/dwn-sdk-js": "0.0.17-unstable.4deecf8-2023.1.17-19-06-19",
```

Comment:
@thehenrytsai should this be just 0.0.17 - as the auto release on main will add that junk at the end automatically?

Reply:
does npm have a mapping from `0.0.17` to the latest unstable?

Reply:
@michaelneale, ideally we should point to stable release as much as possible (e.g. 0.0.17), but we are trying to point to the latest and greatest build right now as `dwn-sdk` is under going a few significant changes.

The version being referenced here is the result of a commit made to the main branch. I have not created an "official release" to trigger the "official release action", because more significant changes are coming.

Reply:
@thehenrytsai oh my mistake, I misread the repo. Ignore my comment!

Reply:
@mistermoe, no, `v0.0.17` would logically map to npm version `0.0.17`